core/src/main/scripts/importer/validateData.py

```diff
@@ -981,6 +981,22 @@ class MutationsExtendedValidator(Validator):
 
     NULL_AA_CHANGE_VALUES = ('', 'NULL', 'NA')
 
+    # MAF values for Variant_Classification column
+    # from https://wiki.nci.nih.gov/display/TCGA/Mutation+Annotation+Format+%28MAF%29+Specification + Unknown:
+    VARIANT_CLASSIFICATION_VALUES = [
+       'Frame_Shift_Del',
+       'Frame_Shift_Ins',
+       'In_Frame_Del',
+       'In_Frame_Ins',
+       'Missense_Mutation',
+       'Nonsense_Mutation',
+       'Splice_Site',
```

Comment:
@sandertan thanks for the extra test on SKCM. If I understand @ckandoth's comments correctly in the ticket you mentioned, `Splice_Region` and `Splice_Site` are useful information and will remain in the files. We might as well just add them to the list and the documentation. I will add them, unless @ckandoth has any concerns against this.